inc/template-functions.php

```diff
@@ -130,6 +130,8 @@ function wmf_get_role_posts( $term_id ) {
 			'post_type'      => 'profile',
 			'fields'         => 'ids',
 			'orderby'        => 'title',
+			'meta_key'       => 'last_name',
+			'orderby'        => 'meta_value',
```

Comment:
I changed to this, so `inc/classes/profiles/class-sorter.php` seems to be no longer needed. Not related to the bug fixed on the PR, but decided to keep it as sorting  directly instead of sorting manually can potentially increase performance.

Reply:
You've got two "orderby" parameters in this query; I think the first "orderby => title" can be removed.